frontend/javascripts/oxalis/model/volumetracing/volumelayer.js

```diff
@@ -304,6 +304,27 @@ class VolumeLayer {
     return map;
   }
 
+  rectangleToCircles(
```

Comment:
Maybe `getRectangleBetweenCircles` or so would be a more descriptive name?

Reply:
That's a way better name, thanks!